lib/helper.js

```diff
@@ -0,0 +1,28 @@
+// Format elapsed vale into a human-readable value, e.g.
+// 2 minutes, 1 second.
+function elapsedFmt (elapsed) {
+  function addUnit (elapsed, div, unit, timeStr) {
+    let modulus = elapsed % div
```

Comment:
You are not reassigning `modulus`, why not using `const`?

Reply:
Good catch. I will fix later. (And somewhere else I see a stray comma when this code style seems to not have those.)